src/otx/algo/detection/utils/assigners/max_iou_assigner.py

```diff
@@ -1,6 +1,10 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-"""Max Iou Assigner implementation from mmdet."""
+# Copyright (c) OpenMMLab. All rights reserved.
+"""Implementations copied from mmdet.models.task_modules.assigners.max_iou_assigner.
```

Comment:
```suggestion
"""Implementation modified from mmdet.models.task_modules.assigners.max_iou_assigner.
```

Reply:
I would suggest using "modified" instead of copied everywhere

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3636/commits/216857e0adc42d9df84e0126f8712ae39d269b84